cypress/fixtures/template_export.json

```diff
@@ -1 +1,608 @@
\ No newline at end of file
+{
```

Comment:
Do you think we can add fixtures/* to the ignore list?
This is because this "template_export.json" and other files are automatically generated, and comes out in minified format

Reply:
I have reverted to the original format and updated the .prettierignore